apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/BatchSpliterator.java

```diff
@@ -51,7 +51,8 @@ public static <T> Stream<List<T>> batches(Stream<T> stream, int batchSize) {
   @Override
   public boolean tryAdvance(Consumer<? super List<T>> action) {
     final List<T> batch = new ArrayList<>(batchSize);
-    for (int i = 0; i < batchSize && base.tryAdvance(batch::add); i++) ;
+    for (int i = 0; i < batchSize && base.tryAdvance(batch::add); i++)
+      ;
```

Comment:
and here as well

Reply:
This is ok..

Reply:
This is trying to be too clever since it's pretty unreadable. Don't need to fix it here but this smells like something that should be adjusted in a way that would remove the semicolon anyway.

Reply:
Personnaly, I think this is vastly better.  Previously the semicolon was hidden at the end of the line and it would easy to mistake whether the loop contained the if statement.